update/upgrade_db.sh

```diff
@@ -73,58 +120,101 @@ fi
 echo "Dumping actual DB"
 TODAY=`date '+%Y%m%d'`
 /usr/bin/pg_dump --host $HOST --port 5432 --username "$USER" --format custom --file "$TODAY""_current_qwat.backup" "$SRCDB"
-#/usr/bin/pg_dump -d "service=$QWATSERVICE" --format custom --file "$TODAY""_current_qwat.backup"
 
-echo "Droping existing qwat_test"
+echo "Dropping existing DB qwat_test"
 /usr/bin/dropdb "$TESTDB" --host $HOST --port 5432 --username "$USER" --no-password
-#/usr/bin/dropdb -d "service=$QWATSERVICETEST"
 
-echo "Creating DB (qwat_test)"
+echo "Creating DB qwat_test"
 /usr/bin/createdb "$TESTDB" --host $HOST --port 5432 --username "$USER" --no-password
-#/usr/bin/createdb -d "service=$QWATSERVICETEST"
 
-echo "Restoring in test DB"
+echo "Restoring current DB in qwat_test"
 /usr/bin/pg_restore --host $HOST --port 5432 --username "$USER" --dbname "$TESTDB" --no-password --single-transaction --exit-on-error "$TODAY""_current_qwat.backup"
-#/usr/bin/pg_restore -d "service=$QWATSERVICETEST" --single-transaction --exit-on-error "$TODAY""_current_qwat.backup"
 
-echo "Applying deltas on $TESTDB:"
+TAB_FILES_POST=()
+echo "Applying deltas on $TESTDB :"
 for f in $DIR/delta/*.sql
 do
     CURRENT_DELTA=$(basename "$f")
-    CURRENT_DELTA_NUM_VERSION=$(echo $CURRENT_DELTA| cut -d'_' -f 2)
-    if [[ $CURRENT_DELTA_NUM_VERSION > $NUMVERSION ]]; then
-        printf "    Processing ${GREEN}$CURRENT_DELTA${NC}, num version = $CURRENT_DELTA_NUM_VERSION\n"
+    CURRENT_DELTA_WITHOUT_EXT="${CURRENT_DELTA%.*}"
+    CURRENT_DELTA_NUM_VERSION=$(echo $CURRENT_DELTA_WITHOUT_EXT| cut -c 7)
+    CURRENT_DELTA_NUM_VERSION_FULL=$(echo $CURRENT_DELTA_WITHOUT_EXT| cut -d'_' -f 2)
+    if [[ $CURRENT_DELTA_NUM_VERSION > $SHORT_LATEST_TAG || $CURRENT_DELTA_NUM_VERSION == $SHORT_LATEST_TAG || $SHORT_LATEST_TAG == '' ]]; then
+        printf "    Processing ${GREEN}$CURRENT_DELTA${NC}, num version = $CURRENT_DELTA_NUM_VERSION ($CURRENT_DELTA_NUM_VERSION_FULL)\n"
         /usr/bin/psql --host $HOST --port 5432 --username "$USER" --no-password -q -d "$TESTDB" -f $f
+
+        # Check if there is a POST file associated to the delta, if so, store it in the array for later execution
+        EXISTS_POST_FILE=$f'.post'
+        if [ -e "$EXISTS_POST_FILE" ]
+        then
+            TAB_FILES_POST+=($EXISTS_POST_FILE)
+        fi
+
+        printf "        Verifying num version conformity - "
```

Comment:
Yes and create an issue for this one. Or we wait until i make the modification (i can make it this week)